console/src/components/Services/Data/TablePermissions/PermissionBuilder/utils.js

```diff
@@ -44,40 +45,103 @@ const boolOperatorsInfo = {
 const columnOperatorsInfo = {
   _eq: {
     type: 'object',
-    PGTypes: ['boolean', 'character', 'dateTime', 'numeric', 'uuid'],
+    PGTypes: [
+      'boolean',
+      'character',
+      'dateTime',
+      'numeric',
+      'uuid',
+      'user_defined',
```

Comment:
Isn't this list the same for every operator? Would it make sense to extract it to the variable, so that another changes would be done just in the one place?

Reply:
It does seem like there can be a better way to represent this info. Will take a stab at it and request a review

Reply:
@beerose Made changes to avoid repetition in the future as well. This was a good catch. Thanks.